yarn.lock

```diff
@@ -6697,7 +6697,7 @@ react-inspector@^1.1.0:
     is-dom "^1.0.5"
 
 "react-instantsearch@file:./packages/react-instantsearch/":
-  version "2.0.0-beta.16"
+  version "2.0.0-beta.17"
```

Comment:
FWIW: Keep this change. It was triggered by "yarn". I then fixed that in the release process so that it's always updated.